documentation/features/cpuset.md

```diff
@@ -0,0 +1,55 @@
+# Cpuset cgroup.
+
+From Kernel documentation:
+
+_" Cpusets provide a mechanism for assigning a set of CPUs and Memory Nodes to
+a set of tasks."_
+
+The Kata agent brings compatibility the cgroup cpuset CPU on the guest side.
+
+The cpuset CPU cgroup will be applied on two events:
+
+- containers creation.
+
+- container update.
+
+When the runtime request to apply cpuset cgroup to the agent, the amout of
```

Comment:
Lines 16-17 suggested rewrite: 
When the runtime requests to apply cpuset cgroup to the agent, the amount of vcpus available might not be the same to the required vcpus in the request.